### ISSUE TITLE
Strange groups layout

### ISSUE THREAD

**Issue by fleith** [olifolkerd/tabulator#1168]
Why does the layout of groups not fill the entire table?

Follow the strange layout:
![image](https://user-images.githubusercontent.com/137489/40745988-69b65b34-642f-11e8-84c2-c05a170e35f7.png)

What I expected is equal when the groups are open, see below:
![image](https://user-images.githubusercontent.com/137489/40746017-82fb361e-642f-11e8-8401-39ebb5ce67d8.png)

The code to reproduce:
```html
<!doctype html>
<html class="no-js" lang="">
<head>
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <title>MRI Suites</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <script type="text/javascript" src="../static/new_js/jquery-3.3.1.min.js"></script>
    <script type="text/javascript" src="../static/new_js/jquery-ui.min.js"></script>
    <link href="../static/new_css/tabulator.min.css" rel="stylesheet">
</head>
<body>
<div id="table"></div>

<script src="../static/new_js/tabulator.min.js"></script>
<script>window.jQuery || document.write('<script src="../static/new_js/jquery-3.3.1.min.js"><\/script>')</script>

<script>
    $("#table").tabulator({
    layout:"fitColumns",
    groupBy: ["cut", "name"],
    groupStartOpen: [true, false],
    columns:[
        {title:"Cut", field:"cut", headerFilter:"input"},
        {title:"Name", field: "name", headerFilter:"input"},
        {title:"Branch", field:"branch", headerFilter:"input" },
        {title:"Platform", field:"platform", headerFilter:"input"},
    ],
});

$("#table").tabulator("setData", [
    {"cut":"a","name":"one","branch":"a","platform":"p"},
    {"cut":"a","name":"two","branch":"a","platform":"p"},
    {"cut":"a","name":"two","branch":"a","platform":"p"},
    {"cut":"a","name":"one","branch":"a","platform":"p"},
    {"cut":"a","name":"three","branch":"a","platform":"p"},
    {"cut":"b","name":"one","branch":"a","platform":"p"},
    {"cut":"b","name":"three","branch":"a","platform":"p"},
    {"cut":"b","name":"three","branch":"a","platform":"p"},
    {"cut":"b","name":"three","branch":"a","platform":"p"},
    {"cut":"c","name":"four","branch":"a","platform":"p"},
    {"cut":"c","name":"four","branch":"a","platform":"p"},
    ]);
</script>

</body>
</html>
```

PS.: Tested on Chrome Version 66.0.3359.181 and Firefox 59.0.3 (64-bit).

Thanks

**Comment by olifolkerd** [comment]
Hey @fleith 

It looks like you have discovered a regression there, it definitely shouldn't be doing that!

I will have a look into releasing a patch for that this weekend. 

Are you using Tabulator v3.5.2?

Cheers

Oli :)

**Comment by fleith** [comment]
Hi @olifolkerd ,
Yes I'm using the Tabulator v3.5.2.

Congrats to the nice project and thanks for looking that!

[],
Alvaro.

**Comment by olifolkerd** [comment]
Hey @fleith 

Sorry for the delay in getting back to you on this one, i have just pushed a fix for this to the master branch and will include it in this evenings patch release.

Cheers

Oli :)

**Comment by fleith** [comment]
Hi @olifolkerd , I found the same problem when I set the field `height`.

You can see at https://jsfiddle.net/4t2wargo/2/ using the last version of Tabulator.

Thanks!

**Comment by olifolkerd** [comment]
Hey @fleith 

Boooo, that sucks...

ok no problem, i will see if i can get to the bottom of it in the 4.0 release.

Thanks for the jsfiddle, that is very helpful.

Cheers

Oli :)

**Comment by olifolkerd** [comment]
Hey @fleith 

I have just pushed a fix for this to the 4.0 release. there should be a beta version of this available in a week.

Cheers

Oli ;)

**Comment by fleith** [comment]
Great! Thanks @olifolkerd
